Docs: Person profiles updates (#8271)
PostHog/posthog.com: contents/docs/data/utm-segmentation.mdx

    classes="rounded"
/>

> > **Note: ** Along with being set on events, initial and latest UTM properties are also added to the [person profile](/docs/data/persons) if it is captured.

Reviewer: ```suggestion
> **Note: ** Along with being set on events, initial and latest UTM properties are also added to the [person profile](/docs/data/persons) when it is created.
```
